clang/lib/CConv/ConstraintVariables.cpp

```diff
@@ -646,13 +646,6 @@ FunctionVariableConstraint::FunctionVariableConstraint(const Type *Ty,
     for (unsigned i = 0; i < FT->getNumParams(); i++) {
       QualType QT = FT->getParamType(i);
 
-      if (InteropTypeExpr *BA =  FT->getParamAnnots(i).getInteropTypeExpr()) {
```

Comment:
Why not handle itypes in the constructor? I'm curious why you deleted these.

Reply:
We do handle this in `PVConstraint`, this code is redundant.

Reply:
OK, great! I love deleting dead/redundant code! :)